src/Scripting/CSharp/CSharpScriptingResources.resx

```diff
@@ -124,21 +124,24 @@
     <value>Copyright (C) Microsoft Corporation. All rights reserved.</value>
   </data>
   <data name="InteractiveHelp" xml:space="preserve">
-    <value><![CDATA[Usage: csi [option] ... [script-file.csx] [script-argument] ...
```

Comment:
><![CDATA[ [](start = 11, length = 9)

Please consider reverting the `<![CDATA[` change if it's not needed.

Reply:
sorry, my bad - it happened automatically in VS - will revert 👍